Library/Homebrew/test/cask/config_spec.rb

```diff
@@ -65,6 +65,10 @@
     it "returns directories explicitly given as arguments" do
       expect(config.explicit[:appdir]).to eq(Pathname("/explicit/path/to/apps"))
     end
+
+    it "returns string of explicit config keys and values" do
+      expect(config.explicit_s).to eq('appdir: "/explicit/path/to/apps"')
+    end
```

Comment:
Might be nice for the test to address the languages case specifically too (either as part of this test or as a separate one).